_test/test_utilities_herbert/test_boxArrayClass.m

```diff
@@ -327,6 +327,75 @@ function test_rand_elmts_position_multiShift (obj)
         end
         
         
+        %--------------------------------------------------------------------------
+        % Test sizes of arrays produced by rand_elmts_position output
+        % Assumes that validate_rand_elmts_position works correctly
+        %--------------------------------------------------------------------------
+        function test_range_elmts_oneShift (obj)
+            % Test range_elmts, duplicating call to range with just
```

Comment:
To a significant extent it is code duplication, but I've left it because (i) the boxArrayClass method uses other functions (e.g.  size_array_stack) and (ii) the internals might change but the test that hand-crafts the reshaping wont. I've generalised internal functioning of methods before and had such tests catch (usually trivial to sort) bugs before.